app/src/main/java/com/humberto/tasky/agenda/presentation/agenda_details/components/AddEventAttendeeDialog.kt

```diff
@@ -0,0 +1,119 @@
+package com.humberto.tasky.agenda.presentation.agenda_details.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.humberto.tasky.R
+import com.humberto.tasky.agenda.presentation.agenda_details.AgendaItemDetails
+import com.humberto.tasky.core.presentation.designsystem.CrossIcon
+import com.humberto.tasky.core.presentation.designsystem.TaskyTheme
+import com.humberto.tasky.core.presentation.designsystem.components.TaskyActionButton
+import com.humberto.tasky.core.presentation.designsystem.components.TaskyDialog
+import com.humberto.tasky.core.presentation.designsystem.components.TaskyTextField
+
+@Composable
+fun AddEventAttendeeDialog(
+    onDismiss: () -> Unit,
+    onAdd: () -> Unit,
+    agendaItem: AgendaItemDetails.Event
+) {
+    if(agendaItem.isAddingAttendee) {
```

Comment:
Ah, here is that boolean value 😅 to manage the visibility of the dialog. Don’t you think the following is more understandable at a glance?

```
if(agendaItem.isAddingAttendee) {
  AddEventAttendeeDialog(/* parameters */)
}
```

Reply:
makes sense :)